docs/src/sources-files.md

```diff
@@ -10,3 +10,32 @@ martin  /path/to/mbtiles/file.mbtiles  /path/to/directory   https://example.org/
 
 You may also want to generate a [config file](config-file.md) using the `--save-config my-config.yaml`, and later edit
 it and use it with `--config my-config.yaml` option.
+
+## PMTiles S3
+
+Martin supports authenticated S3 sources using environment variables.
+
+By default, the credentials try for default profile or environment variables. Environment variables that can to be set:
+
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_SESSION_TOKEN`
+- `AWS_PROFILE` (to specify profile instead of access key variables)
```

Comment:
These could also be done as a table if desired

```suggestion
- `AWS_PROFILE` - to specify profile instead of access key variables
```

Reply:
Given that I don't have other context to put in this table, lets keep it a list for now